BrainPortal/public/javascripts/cbrain.js

```diff
@@ -182,6 +182,19 @@
       $(this).tabs();
     });
 
+    // Find tabs in div with class available_tabs in order to close all overlay dialogs
```

Comment:
This seems awfully complicated to find what to close, no? Why not put a class directly on the elements you want closable and make them easier to find that way?